develop/dev-guide-sample-application-aws-lambda.md

```diff
@@ -0,0 +1,263 @@
+---
+title: Connect to TiDB by using mysql2 in AWS Lambda Function
+summary: This article describes how to build a CRUD application using TiDB and mysql2 in AWS Lambda Function and provides a simple example code snippet.
+---
+
+# Connect to TiDB by using mysql2 in AWS Lambda Function
+
+TiDB is a MySQL-compatible database, [AWS Lambda Function](https://aws.amazon.com/lambda/) is a compute service, and [mysql2](https://github.com/sidorares/node-mysql2) is a popular open-source driver for Node.js.
+
+In this tutorial, you can learn how to use TiDB and mysql2 in AWS Lambda Function to accomplish the following tasks:
+
+- Set up your environment.
+- Connect to your TiDB cluster using mysql2.
+- Build and run your application. Optionally, you can find sample code snippets for basic CRUD operations.
+
+> **Note:**
```

Comment:
```suggestion
> **Note**
```

GitHub defines a set of markdown syntax for considerations, perhaps we can refer to it:

https://github.com/orgs/community/discussions/16925